docker/local_deploy.sh

```diff
@@ -1,3 +1,2 @@
 #!/bin/sh -xe
-kubectl config use-context docker-for-desktop
-ECR_DEPLOY_IMAGE=laalaa_local .circleci/deploy_to_kubernetes development
\ No newline at end of file
+kubectl config use-context docker-for-desktop && ECR_DEPLOY_IMAGE=laalaa_local .circleci/deploy_to_kubernetes development
```

Comment:
Out of curiosity, why were these lines merged?

Reply:
When the context did not exist, the script was not stopping but continuing to execute to whatever context you were currently on.

Reply:
The `e` flag is set on the script. I'm confused as as far as I know,
```
a && b
```
and
```
set -e
a
b
```
should be equivalent. Is that not happening?

Reply:
I changed it to `#!/bin/sh -xe`

However after looking at the majority of the scripts in the docker folder, all the scripts except docker/run_worker.sh use `#!/usr/bin/env bash`. To keep things consist I have changed the script to use `#!/usr/bin/env bash` instead